built-in_analyses.js

```diff
@@ -15,14 +15,18 @@ function my_built_in_analysis(){
   /* Step 4: (optional) If you want to annotate your tableaux with tones,
    * uncomment this block: */
   /*
-  var toneButtons = document.getElementsByName("toneOptions");
+  var toneCheckbox = document.getElementById("annotatedWithTones");
+  toneCheckbox.setAttribute("checked", "checked");
+  var toneButtons = toneCheckbox.parentNode.parentNode.getElementsByTagName("input");
   for(var x = 0; x < toneButtons.length; x++){
+    toneButtons[x].parentNode.setAttribute("style", "display: table-cell");
     if(toneButtons[x].value="addIrishTones_Elfner"){
-      toneButtons[x].setAttribute("checked", true);
+      toneButtons[x].setAttribute("checked", "checked");
     }
     else{
       //we don't want multiple radio buttons to be checked, it gets confusing
-      toneButtons[x].setAttribute("checked", false);
+      //this isn't doing what it is supposed to, I don't know why -Max 10/10/19
```

Comment:
What is it supposed to be doing that it's not doing? It looks like this is intended to ensure that no other tone option can be selected, but with radio buttons, that's automatically the case.

Reply:
Ok, upon further review, I think I see the issue, which is that even though the html shows only one button selected, the objects still both have "checked":"checked". But I think the reason this isn't working is because the value of the Japanese radio button is getting reset

Reply:
Fixed it. See comment above on the conditional.